### ISSUE TITLE
Desktop: Better layout

### ISSUE THREAD

**Issue by umanghome** [flackr/crosswordle#7]
Currently, everything is stacked on top of each other, leading to a vertical scroll when there are a few clues.

![image](https://user-images.githubusercontent.com/5698706/151989852-04738957-b01e-45b7-b562-fa67ed1bfcd5.png)

Maybe we could lay the clues and the game blocks horizontally on Desktop?

![image](https://user-images.githubusercontent.com/5698706/151990156-83fbd197-365d-4d60-97ef-b54f1fefe27e.png)

CSS changes for this^ are on `.game`:

```css
.game {
    grid-area: game;
    justify-self: center;
    width: 100%;
    display: flex;
    align-items: start;
    justify-content: space-around;
    flex-flow: row-reverse;
}
```

but there'd be some responsive breakpoints needed too (or flex-wrap + min-width) for this to work on mobile.

**Comment by flackr** [comment]
Great suggestion! I'll try some options. Right now just using breakpoints it would likely conflict with the puzzle grid trying to be as big as possible - though of course I can change that sizing logic when the breakpoint is hit to reserve width for the clues.

**Comment by flackr** [comment]
Landed a fix with some breakpoints in b79d2ff66bc5f1ad2cebc8508336c58a2a0f8304
